### ISSUE TITLE
Panic with nil field into Params

### ISSUE THREAD

**Issue by samber** [huandu/facebook#148]
I get a panic when calling `Post` method with `nil` attribute.

## OK:

```
session.Post("/me/message", facebook.Params{
	"recipient":     facebook.Params{"id": "1234567890"},
	"sender_action": "typing_on",
})
```

## KO:

```
session.Post("/me/message", facebook.Params{
	"recipient":     facebook.Params{"id": "1234567890"},
	"sender_action": "typing_on",
	"foobar":		 nil,
})
```

## Logs

```
runtime error: invalid memory address or nil pointer dereference
/usr/local/go/src/runtime/panic.go:82 (0x442c90)
     panicmem: panic(memoryError)
/usr/local/go/src/runtime/signal_unix.go:390 (0x442abf)
     sigpanic: panicmem()
/usr/local/go/src/reflect/type.go:3004 (0xa1883a)
     toType: if t == nil {
/go/pkg/mod/github.com/huandu/facebook@v2.3.1+incompatible/params.go:138 (0xa18593)
     Params.Encode: return params.encodeFormUrlEncoded(writer)
/go/pkg/mod/github.com/huandu/facebook@v2.3.1+incompatible/session.go:461 (0xa293ed)
     (*Session).sendPostRequest: mime, err := params.Encode(buf)
/go/pkg/mod/github.com/huandu/facebook@v2.3.1+incompatible/session.go:397 (0xa287d5)
     (*Session).graph: response, err = session.sendPostRequest(graphURL, params, &res)
/go/pkg/mod/github.com/huandu/facebook@v2.3.1+incompatible/session.go:85 (0xa359d3)
     (*Session).Api: return session.graph(path, method, params)
/go/pkg/mod/github.com/huandu/facebook@v2.3.1+incompatible/session.go:95 (0xa3599b)
     (*Session).Post: return session.Api(path, POST, params)
```

**Comment by huandu** [comment]
Fixed by merge request #149.

**Comment by Kivals0711** [comment]
#186
